### DISCUSSION TITLE
OS X- header

### DISCUSSION THREAD

**Discussion by own2pwn** [sbertix/Swiftagram#136] [Q&A]
Do we always have `X-IG-Android-ID` even if the device is iphone / ipad?

https://github.com/sbertix/Swiftagram/blob/e5f113ed71ce82339868c527dc999c5a93ded038/Sources/SwiftagramCrypto/Authentication/Basic/BasicAuthenticator.swift#L183

**Comment by sbertix** [comment]
Hey @own2pwn,  
Good point, it looks like I missed it. 
I'm unfortunately unable to push an update today or tomorrow, but def by the end of the week. Everything still works fine though, I'm assuming they don't actually cross reference it with the other parameters.

**Reply by own2pwn** [reply]
Have you seen any other values? I performed a quick search on github and didn't spot nothing like `ios` nor `iphone`. Maybe official client doesn't use this header at all? But I may just overlooked something.

**Reply by sbertix** [reply]
All GitHub libraries, even **Swift** ones (i.e. **Swiftagram** and [**SwiftyInsta**](https://github.com/them4hd1/swiftyinsta)), are (at least originally) based on the Android API, so I'm assuming that's the reason why that's the only header key popping up. 
I hope to get back at you soon with a better answer and possibly a "fix".

**Reply by own2pwn** [reply]
Hey @sbertix ,
Did you get a chance to take a look at this? I see there's a plenty of new commits in dev branch, should I check it?

**Reply by sbertix** [reply]
I'm definitely planning on fixing this for `5.0.0` but it's not implemented yet. 
The `development` branch is going through a lot of changes, and more are coming in the near future, but you can definitely expect it to be "fixed" on release date.
